python-package/lightgbm/engine.py

```diff
@@ -267,7 +268,12 @@ class CVBooster:
 
     Auxiliary data structure to hold and redirect all boosters of ``cv`` function.
     This class has the same methods as Booster class.
-    All method calls are actually performed for underlying Boosters and then all returned results are returned in a list.
+    All method calls, except for the following methods, are actually performed for underlying Boosters and
+    then all returned results are returned in a list.
+
+    - model_from_string
+    - model_to_string
+    - save_model
```

Comment:
What about [`dump_model()`](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.Booster.html#lightgbm.Booster.dump_model) method?

Reply:
@StrikerRUS 
Currently cvbooster does not override `dump_model`; it returns a list of JSON dictionaries, which I think is consistent with user expectations.

Reply:
Agree, thanks!